src/lib/components/prompt/PromptSuggestion.svelte

```diff
@@ -28,13 +29,23 @@
 	const borderMap: { [key: string]: string } = {
 		ERROR: 'border-l-red-600',
 		ENHANCEMENT: 'border-l-green-600',
-		OPTIMIZATION: 'border-l-yellow-400'
+		OPTIMIZATION: 'border-l-yellow-400',
+		DATA: 'border-l-blue-600'
 	};
 
 	async function accept() {
 		applyingSuggestion = true;
-		const changedPrompt = await acceptSuggestion(suggestion, prompt, userInput);
-		editPrompt(changedPrompt, suggestion.id);
+		if (suggestion.type === 'DATA') {
+			dataGenerationOptions.instruction = suggestion.description
+				.split(':')
+				.slice(1)
+				.join(':')
+				.trim();
+			dataGenerationOptions.show = true;
```

Comment:
can we remove the suggestion once it has been applied? somewhat tricky I know because we're just setting the options but maybe fine to remove once the user clicks apply anyway?

It is definitely weird that it stays even if applied.

Reply:
good idea, you can always regenerate suggestions to get it back